_posts/2024-11-05-engine-news-from-the-latex-project.md

```diff
@@ -72,4 +72,4 @@ engines, and so where possible a move to LuaTeX is suggested.
 The time to move to LuaTeX for new LaTeX documents is here. For existing pdfTeX
 sources, tagging will be available, although it may be more limited than for
 LuaTeX. pdfTeX users can keep their existing sources and will see tagging
-available. XeTeX users really do need to re-work their sources for LuaTeX.
```

Comment:
Don't know what this whitespace change is, perhaps a EOL change? It was [edit](https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files) in-browser.